src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -49,11 +48,9 @@ public void ImportDataForConfirmation()
          var sheets = new Cache<string, DataSheet>();
          foreach (var element in _dataSourceFile.DataSheets.KeyValues)
          {
-            if (!Sheets.Keys.Contains(element.Key))
-            {
-               Sheets.Add(element.Key, element.Value);
-               sheets.Add(element.Key, element.Value);
-            }
+            if (Sheets.Keys.Contains(element.Key)) continue;
```

Comment:
continue on new line or at least empty line. Too easy to miss.
But I am favouring new line for readability